app/vendors/rsa/key.py

```diff
@@ -32,6 +32,7 @@
     or unauthenticated source.
 
 """
+from __future__ import print_function
```

Comment:
Can you revert your changes inside app/vendors? Files in this directory are snapshot of third party libraries installed via PIP, so we don't own the files.